src/modules/configs/schemas/application.py

```diff
@@ -0,0 +1,89 @@
+schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "containers": {
```

Comment:
I think destination and permission objects should be added here too.

Reply:
You can think about (or plan for future via issues/milestones) defining smaller objects and using jsonschema import to re-use existing definitions e.g. `notify` object will be used in all 3 schemas.